_episodes/04-drawing.md

```diff
@@ -135,67 +139,106 @@ The function returns the rectangle as row (`rr`) and column (`cc`) coordinate ar
 > such that it is easier for other people to understand your code.
 {: .callout}
 
-The final section of the program displays the mask we just created:
-
-~~~
-# Display constructed mask
-skimage.io.imshow(mask)
-~~~
-{: .python}
-
-Here is what our constructed mask looks like:
-
-![Maize image mask](../fig/03-maize-mask.png)
-
 > ## Other drawing operations (15 min)
 > 
 > There are other functions for drawing on images, in addition to the 
 > `skimage.draw.rectangle()` function. We can draw circles, lines, text, and
 > other shapes as
 > well. These drawing functions may be useful later on, to help annotate images
-> that our programs produce. Practice some of these functions here. Navigate to
-> the **Desktop/workshops/image-processing/04-drawing-bitwise** directory, and
-> edit the **DrawPractice.py** program. The program creates a black, 800x600 
-> pixel image. Your task is to draw some other colored shapes and lines on the
-> image, perhaps something like this:
-> 
-> ![Sample shapes](../fig/03-draw-practice.jpg)
+> that our programs produce. Practice some of these functions here. 
 > 
-> Circles can be drawn with the `skimage.draw.circle()` function, which takes three
-> parameters: x, y point of the center of the circle, and the radius of the
-> filled circle. There is an optional `shape` parameter that can be supplied to
+> Circles can be drawn with the `skimage.draw.disk()` function, which takes two
+> parameters: the (y, x) point of the center of the circle, and the radius of the
+> circle. There is an optional `shape` parameter that can be supplied to
 > this function. It will limit the output coordinates for cases where the circle
 > dimensions exceed the ones of the image.
 > 
 > Lines can be drawn with the `skimage.draw.line()` function, which takes four
-> parameters: the image to draw on, the (x, y) coordinate of one end of the
-> segment, the (x, y) coordinate of the other end of the segment, and the color
-> for the line.
+> parameters: the (y, x) coordinate of one end of the
+> line, and the (y, x) coordinate of the other end of the line.
 > 
 > Other drawing functions supported by skimage can be found in the 
 > [skimage reference pages](https://scikit-image.org/docs/dev/api/skimage.draw.html?highlight=draw#module-skimage.draw).
 > 
+> First let's make an empty, black image with a size of 800x600 pixels:
+> 
+> ~~~
+> # create the black canvas
+> image = np.zeros(shape=(600, 800, 3), dtype="uint8")
```

Comment:
Perhaps a reference to an earlier episode on the data structure for RGB images would help here to explain the `shape=` parameter?

Reply:
+1 I think a quick reminder of why there is a third dimension would not hurt here.